src/window.rs

```diff
@@ -658,6 +659,39 @@ pub trait WinHandler {
         panic!("release_input_lock was called on a WinHandler that did not expect text input.")
     }
 
+    /// Called when a platform-defined zoom gesture occurs (such as pinching
+    /// on the trackpad).
+    #[allow(unused_variables)]
+    fn zoom(&mut self, delta: f64) {}
+
+    // While the backends transition from mouse events to pointer events, we keep these compatibility
+    // shims.
+
+    #[doc(hidden)]
+    fn mouse_wheel(&mut self, event: &MouseEvent) {
+        self.wheel(&event.clone().into())
+    }
+
+    #[doc(hidden)]
+    fn mouse_move(&mut self, event: &MouseEvent) {
+        self.pointer_move(&event.clone().into())
+    }
+
+    #[doc(hidden)]
+    fn mouse_down(&mut self, event: &MouseEvent) {
```

Comment:
That's a good point. I did it this way to avoid touching the backend code as much as possible (because I know @hardiesoft has some work in progress there and I didn't want to cause conflict. I think at least in the short term that's a more important constraint than a hypothetical glazier user.

Reply:
Makes sense!